code/obj/grille.dm

```diff
@@ -52,6 +52,41 @@
 			name = "catwalk surface" //Or I'll murder you since you are making things ugly on purpose.
 			icon_state = "catwalk_cross" //(Statement does not apply when you actually want to use the other ones.)
 
+		jen // ^^ no i made my own because i am epic
+			name = "maintenance catwalk"
+			icon_state = "catwalk_jen"
+			desc = "This looks marginally more safe than the ones outside, at least..."
+			layer = PIPE_LAYER + 0.01
+			var/turf/my_turf = null
+
+			New()
+				..()
+				my_turf = get_turf_loc(src)
+
+			attack_hand(obj/M, mob/user)
+				return 0
+
+			attackby(obj/item/W, mob/user)
+				if (issnippingtool(W))
+					..()
+				else
+					my_turf.attackby(user.equipped(), user)
```

Comment:
Similarly to the decal, why not just use `src.loc` instead?

Reply:
Agreed, fixing now!